v2/tools/generator/internal/codegen/pipeline/prune_resources_with_lifecycle_owned_by_parent.go

```diff
@@ -69,19 +75,65 @@ func PruneResourcesWithLifecycleOwnedByParent(configuration *config.Configuratio
 	return stage
 }
 
+func checkPrunedEmptyProperties(defs astmodel.TypeDefinitionSet, emptyPrunedProps []astmodel.TypeName) (astmodel.TypeDefinitionSet, error) {
```

Comment:
Is `check` the best verb here? This doesn't really seem like a validation method. Maybe `flagPrunedEmptyProperties`?

Reply:
`Flag` was the word I was looking for. Done!